testament/lib/stdtest/testutils.nim

```diff
@@ -34,3 +35,9 @@ proc greedyOrderedSubsetLines*(lhs, rhs: string): bool =
     if currentPos < 0:
       return false
   return true
+
+const enableRemoteNetworking* = getEnv("NIM_TESTAMENT_REMOTE_NETWORKING") == "1"
```

Comment:
Not sure if hard-coding this into the executable is a good idea, esp. when it's controlled via environment variables at compile time.

Reply:
That's the simplest solution I could find that guarantees the flag propagates; in particular this will propagate to all tests run by testament:
`nim r --putenv:NIM_TESTAMENT_REMOTE_NETWORKING:1 testamen/testament r all`

Reply:
Can't it be an queried at run time? Then the tests would only need to be built once.

Reply:
done